binding/web/template/src/porcupine.ts

```diff
@@ -609,10 +516,7 @@ export class Porcupine implements PorcupineEngine {
     ): Promise<void> {
       const path = arrayBufferToStringAtIndex(memoryBufferUint8, pathAddress);
       try {
-        await pvFileOperationHelper({
-          command: 'file-delete',
-          path: path,
-        });
+        await storage.removeItem(path);
```

Comment:
So according to the types, `removeItem()` could be `async` (indexedDB) or not (localstorage)? I'm not too familiar with Javascript but how will that work out here with the await when it is not async?

Reply:
It basically wraps it into a promise which in this scenario doesn't really affect the function.

Reply:
Sounds good :+1: